### PR TITLE
Setup illwatchers eslint common config

### PR DESCRIPTION
**Pull request by alienalien13**
#2

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
-        "react-app"
+        "react-app",
+        "@illwatchers"
     ],
     "rules": {
-        "react/jsx-filename-extension": [1, {"extensions": [".js"]}],
-        "no-extra-semi": "error",
-        "semi": ["error", "never"],
-        "eol-last": ["error", "never"],
+        "react/jsx-filename-extension": [1, {"extensions": [".js"]}]
```
Comment (maybelinot) [inline .eslintrc.js:8]: Why do we need this rule, when it's already [mentioned](https://github.com/illWatchers/eslint-config/blob/master/index.js#L4) in eslint-config package?

Comment (alienalien13) [reply .eslintrc.js:8]: @maybelinot I excluded it from common config because it is front-end only specific rule

**Review by flashlee — APPROVED** [review]
LGTM :+1:

**Comment by maybelinot** [comment]
@alienalien13 use squash merge for multi-commit PRs, please..

File: .eslintrc.js (hunk 1 above)
Comment (maybelinot) [reply .eslintrc.js:8]: @alienalien13 but it is in [common config](https://github.com/illWatchers/eslint-config/blob/master/index.js#L4) :open_mouth:

Comment (flashlee) [reply .eslintrc.js:8]: @alienalien13 @maybelinot I created issue for excluding this rule from our common config
https://github.com/illWatchers/eslint-config/issues/1

Comment (alienalien13) [reply .eslintrc.js:8]: @flashlee @maybelinot it's my fault. I pushed changes on npm, but forgot to push changes on github.

`use squash merge for multi-commit PRs, please..` I do squash for senseless commits, but it's ok to keep and do not transform history as for me.

Comment (maybelinot) [reply .eslintrc.js:8]: @alienalien13 kk, make sense, lets merge this one

Comment (alienalien13) [reply .eslintrc.js:8]: pushed.
I'm merging this one.

**Pull request merged**